daemon/c_net.c

```diff
@@ -1308,57 +1305,47 @@ c_net_cleanup_interface(c_net_interface_t *ni)
 	}
 }
 
-/**
- * Cleans up the c_net_t struct and shuts down the network interface (in c0's netns).
- */
 static int
-c_net_cleanup_c0(c_net_t *net)
```

Comment:
may comment should be  updated instead of removing?

Reply:
this function is not removed. seems the diff may be missleading